project/assets/scripts/controller/evaluations/evaluations.js

```diff
@@ -1,32 +1,38 @@
-const EvaluationsEvents = function () {
-    this.init = function (redirectToFunction) {
-        setNavEvent('new-evaluation', addEvent.bind(null, 'newEvaluation', redirectToFunction));
-        setNavEvent('logout', addEvent.bind(null, 'login', redirectToFunction));
+(function () {
+    const EvaluationsEvents = function () {
+        this.init = function (redirectToFunction) {
+            setNavEvent('new-evaluation', addEvent.bind(null, 'newEvaluation', redirectToFunction));
+            setNavEvent('logout', addEvent.bind(null, 'login', redirectToFunction));
 
-        const allButtonsObject = document.getElementsByTagName('button');
-        const allButtonsArray = Array.prototype.slice.call(allButtonsObject);
-        allButtonsArray.forEach((element) => {
-            const id = element.getAttribute('id');
-            const detailsBtn = document.getElementById(id);
-            detailsBtn.addEventListener('click', redirectToDetails.bind(null, redirectToFunction, detailsBtn.dataset.name));
-        });
+            const allButtonsObject = document.getElementsByTagName('button');
+            const allButtonsArray = Array.prototype.slice.call(allButtonsObject);
+            allButtonsArray.forEach((element) => {
+                const id = element.getAttribute('id');
+                const detailsBtn = document.getElementById(id);
```

Comment:
Ummm... but... isn't this... like... `element`?!?

Reply:
Aaa..I don't understand what you're saying :D

Reply:
It looks to me as if `element` and `detailsBtn` are the exact same HTML element.

Reply:
Yes, they are, but I do not know the id name that's why I take it with getAttribute and then I take the button having that id.